nixos/tests/postgresql.nix

```diff
@@ -0,0 +1,25 @@
+import ./make-test.nix ({ pkgs, ...} : {
+  name = "postgresql";
+  meta = with pkgs.stdenv.lib.maintainers; {
+    maintainers = [ zagy ];
+  };
+
+  nodes = {
+    master =
+      { pkgs, config, ... }:
+
+      {
+        services.postgresql.enable = true;
+        services.postgresql.initialScript = "/etc/nixos/postgresql-init.sql";
```

Comment:
@zagy Why not write this script directly to the store? Like `pkgs.writeText "postgresql-init.sql" "create role postgres with superuser login createdb;"`. Then you can skip that `cat` invocation.